rct229/ruletest_engine/ruletest_jsons/envelope_tests.json

```diff
@@ -390,7 +390,7 @@
                                                         "tilt": 0,
                                                         "construction": {
                                                             "id": "Construction 1",
-                                                            "u_factor": 2.0328185600815645
+                                                            "u_factor": 0.06305330288229763
```

Comment:
The strict equality test will fail in this case because the U-factor in 90.1 table 3-4 is 0.064 for zone 4 (A,B,C). Could you check again to see if there is any conversion errors or if the starting value is incorrect?

Reply:
There are a number of updates required to be addressed for Rule 5-5 and Rule 5-8 test cases.
1. All test cases for rule 5-5 and 5-8 are missing HVAC system
2. All spaces data group is missing floor_area data (required for zone conditioning category function).
3. rule-5-5-f, rule-5-5-g are missing spaces.

Reply:
Hi Weili,

I believe I fixed the issues you asked. 

- 5-5f and 5-5g were missing spaces in the TCDs for some reason so I added those back in
- HVAC systems were missing due to a formula issue in the spreadsheet. Fixed this.
- Floor area of 250 ft2 was added for all spaces

Reply:
Awesome, thank you Juan. I will merge this PR.